web/api/products.thrift

```diff
@@ -28,6 +28,7 @@ struct ProductConfiguration {
   5: optional DatabaseConnection connection,
   6:          i64                runLimit,
   7: optional bool               isReviewStatusChangeDisabled,
+  8:          string             confidentiality
```

Comment:
Thrift recommends to use optional when extending a struct with new fields:
```suggestion
  8: optional string             confidentiality
```
Please also re-generate the packages after this change.